packages/experimental/RadioGroup/src/Radio/Radio.styling.ts

```diff
@@ -71,34 +71,38 @@ export const stylingSettings: UseStylingOptions<RadioProps, RadioSlotProps, Radi
       ['radioInnerCircleSize', 'radioVisibility', 'radioFill'],
     ),
     labelContent: buildProps(
-      (tokens: RadioTokens) => ({
+      (tokens: RadioTokens, theme: Theme) => ({
         style: {
           alignItems: tokens.labelAlignItems,
           flexDirection: 'column',
           marginRight: tokens.labelMarginRight,
           marginLeft: tokens.labelMarginLeft,
           padding: tokens.labelPadding,
+          paddingRight: 0,
+          ...borderStyles.from(tokens, theme),
         },
       }),
-      ['labelAlignItems', 'labelMarginRight', 'labelMarginLeft', 'labelPadding'],
+      ['labelAlignItems', 'labelMarginRight', 'labelMarginLeft', 'labelPadding', ...borderStyles.keys],
```

Comment:
For my own understanding, this would make it so that borders always apply around the label container instead of around the whole control, even though the whole control accepts mouse events. Does that design make sense?

Reply:
Good question. In the options dialog, native has some limitations with being able to render a focus rect around the whole control. Design came to the conclusion that we want to have the borders always apply around the label container for consistency with native, so this is in line with the updated designs.